packages/berry-cli/sources/cli.ts

```diff
@@ -38,6 +39,25 @@ function runBinary(path: PortablePath) {
   }
 }
 
+async function printAvailableScripts(configuration: Configuration) {
+  const stdout = process.stdout;
```

Comment:
The CLI commands must us the `stdout` parameter passed in parameter to the action callback (see other commands).